docs/1.essentials/2.requirements.md

```diff
@@ -0,0 +1,42 @@
+## Requirements
+
+The hardware requirements are associated with the number of end-users
+(community users) that can be maintained in the node. The minimum number
+of those customers have to be 20. Based on this, the hardware and
+networks requirements are specified in the following sections.
+
+### Hardware Requirements
+
+The hardware requirements are associated with the number of end-users
+(community users) is the following:
+
+-   Up to 10 vCores per end-user.
+
+-   Up to 20 Gb RAM per customer.
```

Comment:
are customers different from end-users? if so, maybe it's better to keep the same wording